.changelog/unreleased/bug-fixes/1016-lowercase-node-id.md

```diff
@@ -0,0 +1,2 @@
+- `[tendermint]` The `tendermint::node::Id` `Display` implementation now prints the hexadecimal string in lowercase ([#971](https://github.com/informalsystems/tendermint-
+  rs/issues/971))
```

Comment:
```suggestion
- `[tendermint]` The `tendermint::node::Id` `Display` implementation now prints the hexadecimal string in lowercase ([#971](https://github.com/informalsystems/tendermint-rs/issues/971))
```

Was this automatically generated by `unclog`? If so I'll have to fix it there. Seems like it was wrapped at the `-` in `tendermint-rs`.

Reply:
Yes, in my experience `unclog` sometimes break lines even in the middle of a URL.

Reply:
Yeah I'm using the `textwrap` crate and I thought I'd configured it to handle URLs properly. Will look into it.

Reply:
I added this manually, thanks for this fixup!